backend/common/config.py

```
@@ -0,0 +1,75 @@
+        env_file=[str(f) for f in _env_files] if _env_files else None,
+        env_file_encoding="utf-8",
+        case_sensitive=False,
+        extra="ignore",
+    )
+
+    # App Config
+    environment: str = "development"
+    # Railway provides DATABASE_URL automatically - Pydantic reads it case-insensitively
+    # Default to SQLite for local development only
+    database_url: str = f"sqlite:///{BASE_DIR / 'app.db'}"
+    secret_key: str = "super-secret-key"
```

Ruff (0.14.8)

Possible hardcoded password assigned to: "secret_key"

(S105)